tests/cli/test_compose_tar_kickstart.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+#####
+#
+# Build tar image and install it using liveimg kickstart command
+#
+#####
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlAssertExists $QEMU_BIN
+        if ! rlCheckRpm httpd; then
+            yum -y install httpd
+        fi
+        systemctl start httpd
+
+        ks_path="/var/www/html/ks-tar.cfg"
+        tmp_dir=$(mktemp -d /tmp/composer.XXXXX)
+        ssh_key_dir=$(mktemp -d /tmp/composer-ssh-keys.XXXXXX)
+
+        rlRun -t -c "ssh-keygen -t rsa -N '' -f $ssh_key_dir/id_rsa"
+        pub_key=$(cat $ssh_key_dir/id_rsa.pub)
+
+        bp_name="test-tar"
+        blueprint="$bp_name.toml"
+        cat > $blueprint << __EOF__
+name = "$bp_name"
+description = "tar image test"
+version = "0.0.1"
+modules = []
+
+[[packages]]
+name = "openssh-server"
+version = "*"
+
+# sudo and auditd are needed for checks performed on the installed image instance
+[[packages]]
+name = "sudo"
+version = "*"
+
+[[packages]]
+name = "audit"
+version = "*"
+
+[[groups]]
+name = "anaconda-tools"
+version = "*"
```

Comment:
Also missing package kernel which is stated as required in the description of this test case. The variant which was on master branch was missing anaconda-tools+kernel and was running into some issues. Retest is still in progress (#887)

Reply:
The test works for me as it is, without `kernel` package. IIRC the need for `anaconda-tools`/`kernel` differs across the main branches, doesn't it? Also most probably `audit` should go away if #871 is backported, as Brian suggests.